account_consolidation/wizard/consolidation_base.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2011-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, _, api
+from odoo.exceptions import UserError
+
+
+class AccountConsolidationBase(models.AbstractModel):
+    _name = 'account.consolidation.base'
+    _description = 'Common consolidation wizard. Intended to be inherited'
+
+    @api.model
+    def _default_company(self):
+        return self.env['res.company']._company_default_get()
+
+    @api.model
+    def _default_subsidiaries(self):
+        return self._default_company().child_ids
+
+    company_id = fields.Many2one(
+        'res.company',
+        default=_default_company,
+        domain=[('is_consolidation', '=', True)],
+        string='Company',
+        required=True)
+
+    subsidiary_ids = fields.Many2many(
+        'res.company',
+        'account_conso_comp_rel',
+        'conso_id',
+        'company_id',
+        default=_default_subsidiaries,
+        string='Subsidiaries',
+        required=True)
+
+    def check_subsidiary_mapping(self, conso_holding_accounts, subsidiary):
+        """
```

Comment:
Not a requirement, but would be nice to follow [PEP guidelines for docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings): shorten 1st line w/ period at the end. One blank line, then the rest ;)